src/features/finance/FoodList.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
```

Comment:
Linters represent coding styles that you share with your team. As much as possible, kindly avoid disabling them in line as they most of the time also enforce best coding practices